### ISSUE TITLE
Does not work, provide pictures please.

### ISSUE THREAD

**Issue by superpawko** [rkosov/Lingua-Libre-and-Forvo-Audio-Downloader#1]
Tools  → Addons  → Forvo Audio Downloader.   There is no such thing 

my config: 

{
    "Germany": {
        "fields": {
            "EN_WORD": "EN_WORD_AUDIO"
        },
        "language": [
            "de",
            "deu"
        ],
        "note type": "Multikurs"
    },
    "default": {}
}

when i go Tools - download audio 
![image](https://user-images.githubusercontent.com/11931157/171846914-cdfa8427-067a-41f2-b6ad-47d175cd0eae.png)

Using this button: 
![image](https://user-images.githubusercontent.com/11931157/171847268-d3f0839d-9352-4cfd-b1c6-7a112499c5ef.png)

I got this error: 
`Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.53 (96bacf79) Python 3.9.7 Qt 5.15.2 PyQt 5.15.5
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2022-06-03 13:20:56

Caught exception:
Traceback (most recent call last):
  File "aqt.webview", line 42, in cmd
  File "aqt.webview", line 149, in _onCmd
  File "aqt.webview", line 619, in _onBridgeCmd
  File "decorator", line 232, in fun
  File "anki.hooks", line 89, in decorator_wrapper
  File "anki.hooks", line 86, in repl
  File "C:\Users\Pawel\AppData\Roaming\Anki2\addons21\799647424\main.py", line 183, in on_bridge_cmd
    _old(editor, command)
  File "decorator", line 232, in fun
  File "anki.hooks", line 89, in decorator_wrapper
  File "anki.hooks", line 86, in repl
  File "C:\Users\Pawel\AppData\Roaming\Anki2\addons21\24411424\custom_shortcuts.py", line 440, in cs_captureBridgeToggleSticky
    return _old(self,cmd)
  File "aqt.editor", line 467, in onBridgeCmd
  File "C:\Users\Pawel\AppData\Roaming\Anki2\addons21\124265771\audio_downloader.py", line 807, in button_pressed
    load_ll_database()
  File "C:\Users\Pawel\AppData\Roaming\Anki2\addons21\124265771\audio_downloader.py", line 279, in load_ll_database
    ll_database = json.load(f)
  File "json", line 293, in load
  File "json", line 346, in loads
  File "json.decoder", line 337, in decode
  File "json.decoder", line 353, in raw_decode
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1063394 column 13 (char 32616310)
`

**Comment by rkosov** [comment]
It seems that your Lingua Libre json file became corrupt. I added some code to handle this situation. Please update the addon and recheck.

**Comment by superpawko** [comment]
Yes indeed now it works.

I want to use it in batch, when I used Download it started downloading it for too many so I tried to limit it. I made new deck.  I have set filter to deck name, and now I have different error: 

here is config: 
```
{
    "Germany": {
        "deck": "2do",
        "fields": {
            "EN_WORD": "EN_WORD_AUDIO"
        },
        "language": [
            "de",
            "deu"
        ],
        "note type": "Multikurs"
    },
    "default": {}
}
```

![image](https://user-images.githubusercontent.com/11931157/171911546-de309799-2695-4c20-af73-e1bf0806953e.png)
![image](https://user-images.githubusercontent.com/11931157/171911611-db2fd28f-7d7e-4c54-8a38-3c27002e5efa.png)
